x-pack/plugins/siem/public/components/page/hosts/host_overview/index.tsx

```diff
@@ -49,7 +59,9 @@ export const HostOverview = pure<HostSummaryProps>(({ data, loading }) => {
     [
       {
         title: i18n.HOST_ID,
-        description: data.host ? hostIdRenderer(data.host) : getEmptyTagValue(),
+        description: data.host
```

Comment:
by reviewing this code, I was wondering if we can make this function cleaner like that 
```
interface HostIdRendererTypes {
  host: HostEcsFields;
  ipFilter?: string;
  noLink: boolean;
}

export const hostIdRenderer = ({
  host,
  ipFilter,
  noLink = false,
}: HostIdRendererTypes): React.ReactElement =>
  host.id && host.ip && (ipFilter == null || host.ip.includes(ipFilter)) ? (
    <>
      {host.name && host.name[0] != null ? (
        <DefaultDraggable id={`${IpOverviewId}-host-id`} field="host.id" value={host.id[0]}>
          {noLink ? (
            <>{host.id}</>
          ) : (
            <HostDetailsLink hostName={host.name[0]}>{host.id}</HostDetailsLink>
          )}
        </DefaultDraggable>
      ) : (
        <>{host.id}</>
      )}
    </>
  ) : (
    getEmptyTagValue()
  );
```

Reply:
Awesome, thanks!